src/components/views/host_signup/HostSignupContainer.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2021 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, {useState} from 'react';
```

Comment:
inconsistent quotes

Reply:
Which is the preferred btw? I see both in the code base.

Reply:
We care more about consistency than one or the other; 

The code style doesn't specify either but only uses `"` by example https://github.com/matrix-org/matrix-react-sdk/blob/master/code_style.md